devkit/bootstrap-unix.sh

```diff
@@ -199,22 +255,24 @@ fi
 # Clone salt states
 echo Cloning the development kit repository
 tmpdir=$(mktemp -d -t pavedroad.XXXXXX 2>/dev/null)
-git clone ${branch} https://github.com/pavedroad-io/pavedroad.git ${tmpdir}
+rm -rf ${tmpdir}
+${usersudo} git clone ${branch} https://github.com/pavedroad-io/pavedroad.git ${tmpdir}
 
 # Apply salt states
 echo Installing the development kit
-saltdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" &>/dev/null && pwd)
-${sudo} ${tmpdir}/devkit/apply-state.sh ${debug}
-mv ${tmpdir} ${saltdir}
+${usersudo} ${sudo} ${tmpdir}/devkit/apply-state.sh ${debug}
+# ${usersudo} ${sudo} ${tmpdir}/devkit/check-state.sh docker
+mv ${tmpdir} ${boothome}
+mv pr-root ${boothome}/$(basename ${tmpdir})/devkit
 
 # Temporary fix until permanent fix TBD in salt states
-if ${chown}; then
-    homedir=$(eval echo ~$(id -un))
-    ${sudo} chown -R $(id -un):$(id -gn) ${homedir}
+if [ ${chown} ] ; then
+    ${sudo} chown -R ${bootuser}:${bootgroup} ${boothome}
 fi
 echo Development kit installation complete
 
-if command -v xdg-open >& /dev/null; then
+if [ ${DISPLAY} ] && command -v xdg-open >& /dev/null; then
     echo Opening the getting started page for the development kit
+    export BROWSER=/usr/bin/firefox
```

Comment:
probably should be a grain instead of hard coding firefox

Reply:
As the devkit installs firefox it is the only browser guaranteed to be there for any new system across supported OSes and/or versions